fontforge/ufo.c

```diff
@@ -3409,14 +3301,7 @@ static void MakeKerningClasses(SplineFont *sf, struct ff_glyphclasses *group_bas
   // This silently ignores already extant groups for now but avoids duplicates unless group_base has internal duplication.
   int left_count = 0, right_count = 0, above_count = 0, below_count = 0;
   int left_start = 0, right_start = 0, above_start = 0, below_start = 0;
-#ifdef FF_UTHASH_GLIF_NAMES
-    struct glif_name_index _class_name_hash;
-    struct glif_name_index * class_name_hash = &_class_name_hash; // Open the hash table.
-    memset(class_name_hash, 0, sizeof(struct glif_name_index));
-    HashKerningClassNames(sf, class_name_hash);
```

Comment:
This was a little bit of a mystery; it does this hashing, but then never uses the hash later on. I'm pretty sure `HashKerningClassNames` has no side effects on `sf` too.

Reply:
The plan was that this function would check old names and not add duplicates, but this seems not to have happened, probably because it was unnecessary in the current use case. So my preceding comment about ignoring extant groups is probably wrong.